presto-main/src/main/java/com/facebook/presto/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -1553,4 +1553,22 @@ public static ExpressionAnalyzer createWithoutSubqueries(
             throw new SemanticException(errorCode, node, message);
         }, session, ImmutableMap.of(), parameters, isDescribe);
     }
+
+    public static ExpressionAnalyzer createWithoutSubqueries(
+            FunctionRegistry functionRegistry,
+            TypeManager typeManager,
+            Session session,
+            Map<Symbol, Type> symbolTypes)
+    {
+        return new ExpressionAnalyzer(
+                functionRegistry,
+                typeManager,
+                node -> {
```

Comment:
doesn't any of the existing error codes fits? Who used `createWithoutSubqueries` with semantic error code?